### ISSUE TITLE
JAX

### ISSUE THREAD

**Issue by janosh** [simple-icons/simple-icons#8357]
### Brand Name

JAX

### Website

https://jax.readthedocs.io

### Popularity Metric

https://github.com/google/jax has ~22k GH stars

https://www.similarweb.com/website/jax.readthedocs.io/#traffic-sources

### Official Resources for Icon and Color

https://github.com/google/jax/blob/97a5719fcb40af7231b5f803f965063538282f8e/images/jax_logo.svg

### Additional Comments

![JAX Logo](https://raw.githubusercontent.com/janosh/awesome-normalizing-flows/main/assets/jax.svg)

### Contributing

Yes

**Comment by janosh** [comment]
Related icon to PyTorch and Tensorflow which are already in this collection.

**Comment by PeterShaggyNoble** [comment]
See https://github.com/simple-icons/simple-icons/issues/13048#issuecomment-2802155356

**Comment by PeterShaggyNoble** [comment]
Closing due to lack of permission. If anyone wants to try contacting them to gain permission then please leave a comment and we'll reopen this issue.
